test/powershell/Modules/Microsoft.PowerShell.Utility/MarkdownCmdlets.Tests.ps1

```diff
@@ -546,4 +516,98 @@ bool function()`n{`n}
             }
         }
     }
+
+    Context "Hosted PowerShell scenario" {
+
+        It 'ConvertFrom-Markdown gets expected output when run in hosted powershell' {
+
+            try {
+                $pool = [runspacefactory]::CreateRunspacePool(1, 2, $Host)
+                $pool.Open()
+
+                $ps = [powershell]::Create()
+                $ps.RunspacePool = $pool
+                $ps.AddScript( {
+                        try {
+                            $output = '# test' | ConvertFrom-Markdown
+                            $output.Html.trim()
+                        } catch {
+                            throw $_
+                        }
+                    })
+
+                $output = $ps.Invoke()
+
+                $output | Should -BeExactly '<h1 id="test">test</h1>'
+            } finally {
+                $ps.Dispose()
+            }
+        }
+
+        It 'Get-MarkdownOption gets default values when run in hosted powershell' {
+
+            try {
+                $ps = [powershell]::Create()
+                $ps.AddScript( {
+                    Get-MarkdownOption -ErrorAction Stop
+                })
+
+                $options = $ps.Invoke()
+
+                $options.Header1 | Should -BeExactly "[7m"
```

Comment:
I'd put on first place to get more clear message if `Get-MarkdownOption` fail:
```suggestion
                $options | Should -Not -Be $null
```

Reply:
Fixed.